src/FMI2/cconst.jl

```diff
@@ -121,6 +145,13 @@ To simplify porting, no C types are used in the function interfaces, but the ali
 All definitions in this section are provided in the header file “fmi2TypesPlatform.h”.
 """
 const fmi2True = fmi2Boolean(true)
+
```

Comment:
this docstring ssems to be at the wrong place (between fmi2True and fmi2False?)

Reply:
thats because that is the docstring for fmi2False. There is the same for fmi2True just above. As both are exported, both need docstrings.

Reply:
fixed